### PR TITLE
Update pin-configuration.md

### PR DESCRIPTION
**Pull request by JamesH65**
Added missing pin_* sections

### CODE REVIEW

File: configuration/pin-configuration.md [hunk 1]
```diff
@@ -30,17 +30,22 @@ The `dt-blob.bin` is used to configure the binary blob (VideoCore) at boot time.
...
+ - **pins_rev2** Rev2 pin setup. This includes the additional codec pins on P5.
+ - **pins_bplus1** Model B+ rev 1.1, including the full 40pin connector.
+ - **pins_bplus2** Model B+ rev 1.2, swapping the low-power and lan-run pins.
+ - **pins_aplus** Model A+, lacking Ethernet.
+ - **pins_2b1** Pi 2 Model B rev 1.0; controls the SMPS via I2C0.
+ - **pins_2b2** Pi 2 Model B rev 1.1; controls the SMPS via software I2C on 42 and 43.
+ - **pins_3b1** Pi 3 Model B rev 1.0
+ - **pins_3b2** Pi 3 Model B rev 1.2
+ - **pins_pi0** The Pi Zero
+ - **pins_pi0w** The PiZero wireless
```
Comment (pelwell) [inline configuration/pin-configuration.md]: "Pi Zero Wireless"?

**Comment by AisforAstronaut** [comment]
Changed to 'Pi Zero W'. Pushed back for final check.

File: configuration/pin-configuration.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ The `dt-blob.bin` is used to configure the binary blob (VideoCore) at boot time.
  - **pins_3b1** Pi 3 Model B rev 1.0
  - **pins_3b2** Pi 3 Model B rev 1.2
  - **pins_pi0** The Pi Zero
- - **pins_pi0w** The PiZero wireless
+ - **pins_pi0w** The Pi Zero W
```
Comment (pelwell) [inline configuration/pin-configuration.md]: But https://www.raspberrypi.org/products/pi-zero-wireless/ - one of them ought to change.

Comment (JamesH65) [inline configuration/pin-configuration.md]: @ebenupton  Anyone know what the official term is?

**Comment by AisforAstronaut** [comment]
The Comms style guide would suggest 'Raspberry Pi Zero W'. That's not
confirmed, but it would follow the style for other models.

**Comment by JamesH65** [comment]
Hmm, guess we need to wait for confirmation from Eben, then alter the appropriate docs for consistency.

**Comment by JamesH65** [comment]
Pi Zero W it is!

**Pull request merged**